README.md

```diff
@@ -65,8 +109,8 @@ None.
 
 | Variable | Description | Default | Required |
 |----------|-------------|---------|----------|
-| cdm_certificates_certificate_object_names | A list of AWS S3 objects corresponding to the CDM certificates. | `["cert_US_CISA_0_Root_CA.crt","cert_US_CISA_1_Issuing_CA_reissued.crt"]` | No |
-| cdm_certificates_third_party_bucket_name | The name of the AWS S3 bucket where third-party software is located. | `cisa-cool-third-party-production` | Yes |
+| cdm\_certificates\_certificate\_object\_names | A list of AWS S3 objects corresponding to the CDM certificates. | `["cert_US_CISA_0_Root_CA.crt","cert_US_CISA_1_Issuing_CA_reissued.crt"]` | No |
+| cdm\_certificates\_third\_party\_bucket\_name | The name of the AWS S3 bucket where third-party software is located. | None | Yes |
```

Comment:
I think our unwritten standard here has been `n/a`.  It's what I've been using in these recent PRs because I saw we had previously done it elsewhere:
```suggestion
| cdm\_certificates\_third\_party\_bucket\_name | The name of the AWS S3 bucket where third-party software is located. | n/a | Yes |
```